program/tests/duplicate_block_proof.rs

```diff
@@ -60,6 +61,16 @@ async fn setup_clock(context: &mut ProgramTestContext) {
     context.set_sysvar(&new_clock);
 }
 
+async fn increment_clock(context: &mut ProgramTestContext, increment: Epoch) {
```

Comment:
nit: can this be called `increment_clock_epoch` to be totally clear? A lot of clock-setting functions typically work with the slot, and not the epoch, so this will make the function unambiguous